genai-perf/genai_perf/config/input/config_command.py

```diff
@@ -68,15 +72,23 @@ def __init__(
         self.output = ConfigOutput()
         self.tokenizer = ConfigTokenizer()
 
-        self._parse_yaml(user_config, create_template)
+        self._parse_yaml(user_config, skip_inferencing_and_checking)
 
     ###########################################################################
     # Top-Level Parsing Methods
     ###########################################################################
+    def infer_and_check_options(self) -> None:
+        """
+        Infers and checks the configuration options.
+        """
+        self._infer_settings()
+        self._check_for_illegal_combinations()
+        self._check_profile_export_file()
+
     def _parse_yaml(
         self,
         user_config: Optional[Dict[str, Any]] = None,
-        create_template: bool = False,
+        skip_inferencing_and_checking: bool = False,
```

Comment:
what is "skip inferencing and checking"?

Reply:
It is also used when creating a blank config file that will be populated via the CLI (as DavidY has noted in his comment below).